pkgs/build-support/go/module.nix

```diff
@@ -2,14 +2,10 @@
 
 { name ? "${args'.pname}-${args'.version}"
 , src
-, buildInputs ? []
 , nativeBuildInputs ? []
 , passthru ? {}
 , patches ? []
 
-# Go linker flags, passed to go via -ldflags
-, ldflags ? []
```

Comment:
uh? don't we use that everywhere?

Reply:
only at build time

https://github.com/NixOS/nixpkgs/blob/d30264c2691128adc261d7c9388033645f0e742b/pkgs/build-support/go/module.nix#L229

Reply:
oh is that how it works...